charts/images.yaml

```diff
@@ -4,10 +4,14 @@
 # the respective tag can be used. The syntax must be as described in the
 # Masterminds/semver package: https://github.com/Masterminds/semver#hyphen-range-comparisons.
 images:
-# Gardenlet
+# Gardener components
 - name: gardenlet
   sourceRepository: github.com/gardener/gardener
   repository: eu.gcr.io/gardener-project/gardener/gardenlet
+- name: gardener-seed-admission-controller
+  sourceRepository: github.com/gardener/gardener
+  repository: eu.gcr.io/gardener-project/gardener/seed-admission-controller
+  tag: "v1.2.0-dev"
```

Comment:
I'm confused, we discussed this on the phone two days ago. The reason is that for non-released versions the Gardenlet thinks its own version is `vX.Y.Z-dev`, and for this no image tag exists. Hence, if you'd check out the `master` branch (or any non-released commit) then it won't work. Hence, we decided together that specifying the version here is the best approach for now. After the `v1.2` release the version here will always be up-to-date with the latest released version (see `prepare_release` CI script).

Reply:
Sorry, I must have missed that discussion on the phone, there was a bad connection during the meeting 😞
Anyways, got it. Thanks for explaining.

Reply:
😄 Alright, np. :)

Reply:
I changed my mind. I'll change the code in a way that the `charts/images.yaml` does not contain the tag by default. In this case, the Gardenlet will deploy GSAC with its current version (if it is not a dev version). If the current version is a dev version then it will deploy it with the last stable version.